Analyzer/test/MyAnalysis.C

```diff
@@ -51,12 +67,19 @@ template<typename Analyze> void run(std::set<AnaSamples::FileSummary> vvf,
         // Define classes/functions that add variables on the fly
         std::shared_ptr<RunTopTagger> rtt;
         if ( !isSkim ) rtt = std::make_shared<RunTopTagger>();
-        RunFisher runFisher;
+        RunFisher runFisher("v3",myVarSuffix);
```

Comment:
Is the RunFisher constructor able to take two arguments?

Reply:
I just checked and the current version does not. 
https://github.com/StealthStop/Framework/blob/master/Framework/include/RunFisher.h#L194